components/beta-bar/beta-bar.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="beta-bar">
+  <div class="container beta-bar__inner">
+    <h3 class="beta-bar__badge">Beta</h3>
+    <p class="beta-bar__desc u-fs-s">This is a new service – your <a href="#">feedback</a> will help us improve it.</p>
+  </div>
+</div>
+
```

Comment:
I think we could handle this without some of the new classes and adding a modifier to the existing `grid__col--flex` class. Have a look below:

```suggestion
<div class="beta-bar">
  <div class="container">
    <div class="grid grid--flex">
      <div class="grid__col grid__col--flex grid__col--vertical-center">
        <h3 class="beta-bar__badge">Beta</h3>
        <p class="u-fs-s u-mb-no">This is a new service – your <a href="#">feedback</a> will help us improve it.</p>
      </div>
    </div>
  </div>
</div>
```

We would need to add `grid__col--vertical-center` to `align-items: center`.

Reply:
Thats a good suggestion... I'll push something up that utilises the flex grid more for this.